acceptancetests/deploy_stack.py

```diff
@@ -1043,6 +1104,27 @@ def existing_booted_context(self, upload_tools, **kwargs):
         except LoggedException:
             sys.exit(1)
 
+    @contextmanager
+    def existing_context(self, upload_tools, env_name, **kwargs):
+        env_name = env_name.split('-temp-env')[0]
```

Comment:
See comment above re: using replace.

Why with the need to remove '-temp-env'? If it's required are we able to nip it i nthe bud in a single location and not rename at random points in the code?

Actually while typing this I notice env_name isn't actually used here at all?